### ISSUE TITLE
Run the load every night

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-chado#550]
Add a cron job that starts the load automatically.

**Comment by ValWood** [comment]
Seems we get about 100 times more  frequent loads than updates do can deploy, so deprioritize until we have new web pages and need to update nightly ;)

**Comment by kimrutherford** [comment]
I've finally got around to setting this up because I was tired of starting loads manually.  I still need to manually point Canto at the new load, but that's quick.

The load now starts at 11:45pm UK time each night and should finish a little before 9am each morning.  That avoids most of the UK working day and most of the North American working.  Not so great for Asia though.  Running the load in the background doesn't seem to cause Canto to go slow but I'll keep an eye on it.

I haven't changed anything else so the results should appear in: https://curation.pombase.org/dumps/latest_build/

It's probably good to check the date on files before you do any log checking just to make sure the load hasn't completely failed.  That doesn't happen too often these days.

I'll close this issue tomorrow I've checked it all works OK.
